README.md

```diff
@@ -22,114 +22,4 @@ limitations under the License.
 >
 > ##### LATEST RELEASE
```

Comment:
Isn't this line redundant for "warning" message?

Reply:
I was following the format done [before](https://github.com/triton-inference-server/model_analyzer/commit/7e2cf8d34a39b1aec55ed40751ab20893b3efcd8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5). Please let me know if this needs to be updated.

Reply:
@fpetrini15  for viz

Reply:
I also followed the format from [before](https://github.com/triton-inference-server/model_analyzer/pull/897/files). Unfortunately, it seems like we carry over issues. Hopefully, we can dedicate some time soon to streamlining this process.